### ISSUE TITLE
Fail to import

### ISSUE THREAD

**Issue by pcchen** [kaihsin/Cytnx#188]
With recent commits (openblas problem fixed), I am able to compile and install, but when I try to import cytnx, I encounter following error:

````
>>> import sys
>>> sys.path.insert(0,'/usr/local/cytnx')
>>> import cytnx
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: arg(): could not convert default argument into a Python object (type not registered yet?). Compile in debug mode for more information.
````

**Comment by kaihsin** [comment]
Is this from dev-master?

**Comment by pcchen** [comment]
Yes it is from dev-master.

**Comment by kaihsin** [comment]
This is due to improper python binding with new dtype arguments on cytnx.random. Fixed in [eabfda8](https://github.com/kaihsin/Cytnx/commit/eabfda84ef211a419a2ad82c7dfc309f6ff20d2f)
